source/sync/data-model/development-mode.txt

```diff
@@ -12,15 +12,15 @@ Development Mode
    :depth: 1
    :class: singlecol
 
-You can design schemas for your {+app+} directly in client code. The objects 
+You can design schemas for your app directly in client code. The objects 
 and relationships that you define in your application become the server-side 
-document schema based on your SDK models. :ref:`{+client-sdks+} <realm-sdks>` 
+document schema based on your SDK models. :ref:`Realm SDKs <realm-sdks>` 
 give you language-specific tools to define and work with these objects.
 
 You can infer schemas from your client object models by enabling development 
-mode. Development mode is an Atlas Device Sync configuration setting for your {+app+}.
-When you enable Development Mode, your {+app+} can create or update your 
-schema by reading your synced {+realm+} files.
+mode. Development mode is an Atlas Device Sync configuration setting for your app.
+When you enable Development Mode, your app can create or update your 
+schema by reading your synced realm files.
```

Comment:
imo would be clearer to say 'Realm Database instance'. i don't think including the notion that a realm (in the client) is a file is useful/necessary in the server-side docs. rather fact that it's a file is just an implementation detail

Reply:
ditto rest of page

Reply:
for rest of page, rather than 'Realm Database instance', you can just say 'realm'